pkg/core/manager/insight/resource.go

```diff
@@ -34,17 +35,19 @@ func (i *InsightManager) GetResource(
 	if err != nil {
 		return nil, err
 	}
+	secret, err := client.DynamicClient.Resource(resourceGVR).Namespace(resourceGroup.Namespace).Get(ctx, resourceGroup.Name, metav1.GetOptions{})
```

Comment:
It is not sure here is secret, the variable name can be changed to a universal name.